packages/framework/src/Resources/styles/admin/components/box/overview.less

```diff
@@ -60,6 +61,14 @@
                     transform: rotate(180deg);
                 }
             }
+
+            &--same {
+                background-color: @box-overview-color-same;
+
+                .svg {
+                    display: none;
```

Comment:
what if we just not generate it via twig???

Reply:
Well then you would need some conditions in current macro ... I prefer this way